apps/website/.vuepress/theme/global-components/DocDemo.vue

```diff
@@ -16,12 +16,8 @@ export default {
 
 <style lang="scss">
 .demo-wrapper {
-  // @TODO Why did we have these? They break tooltips
```

Comment:
to contain the demos of modals and such. if those are still working as expected, 👍 

if not, you may need to make a second demo wrapper for them...

Reply:
I went through the angular demos. They should all be visible w/o any overflow so I believe that this stuff isn't needed any longer.